src/main/java/org/opentripplanner/updater/alerts/AlertsUpdateHandler.java

```diff
@@ -95,6 +95,12 @@ private void handleAlert(String id, GtfsRealtime.Alert alert) {
             if (informed.hasRouteId()) {
                 routeId = informed.getRouteId();
             }
+
+            String direction = null;
+            if (informed.hasTrip() && informed.getTrip().hasDirectionId()) {
+                direction = String.valueOf(informed.getTrip().getDirectionId());
```

Comment:
In that case I kind of wonder why it's being stored as a String here... I may just be overlooking something blindingly obvious. ;)